README.md

```diff
@@ -293,13 +309,13 @@ Objective-C:
 ```
 The response can be passed to the `processResponse` method as a parameter as shown above. This method returns the response to the SDK to process - attaching any capture, track or optimize instructions to the Interaction.
 
-*Note:* 
-- This will send a `POST` request to ONE. 
+*Note:*
+- This will send a `POST` request to Thunderhead ONE or Salesforce Interaction Studio.
 - When sending Interaction requests programmatically please ensure the Interaction starts with a `/` and only contains letters, numbers and/or dashes.
 
 ### Retrieve a response for a UIViewController instance
 
-You can retrieve a response for an automatically triggered specific Interaction request by making your object conform to the protocol `OneInteractionResponseDelegate`. Your object might be an instance of `UIViewController` or any other class. Follow the instructions below in order to set up this functionality depending on your object’s class. 
+You can retrieve a response for an automatically triggered specific Interaction request by making your object conform to the protocol `OneInteractionResponseDelegate`. Your object might be an instance of `UIViewController` or any other class. Follow the instructions below in order to set up this functionality depending on your object’s class.
```

Comment:
"You can retrieve a response for an automatically triggered specific Interaction request"

Doesn't mean anything to me is this supposed to mean 

"You can retrieve a response for a specific, automatically triggered Interaction request"